containers/datalab/Dockerfile.in

```diff
@@ -97,7 +97,8 @@ RUN mkdir -p /datalab/web && \
         node-uuid@1.4.3 \
         bunyan@1.4.0 \
         socket.io@1.3.6 \
-        tcp-port-used@0.1.2 && \
+        tcp-port-used@0.1.2 \
+        node-cache && \
```

Comment:
I also expected an updated package.json file. Please run

    npm install node-cache --save

to have that be updated, which should also make it easy to get the version #.